package.json

```diff
@@ -13,7 +13,6 @@
   },
   "dependencies": {
     "@auth0/auth0-react": "2.2.4",
-    "@aws-amplify/ui-react": "^5.0.4",
```

Comment:
Because `Authenticator` in `@aws-amplify/ui-react@5` changed from `AmplifyAuthenticator` in `@aws-amplify/ui-react@1`, and no longer provides a "Log In with AWS" link to perform an oauth-based redirect login using the hosted UI, this library is simply removed and redirection to the hosted UI occurs immediately upon load of `AppCognito.tsx` without needing to click "Log in with AWS" which had been provided by `AmplifyAuthenticator` back at version 1.